src/app.types.tsx

```diff
@@ -68,25 +94,19 @@ export interface CatalogueItemPropertyResponse {
   unit: string;
 }
 
-export interface CatalogueItem {
+export interface CatalogueItem extends CatalogueItemDetails {
   catalogue_category_id: string;
-  name: string | undefined;
-  description: string;
   properties: CatalogueItemPropertyResponse[];
   manufacturer: CatalogueItemManufacturer;
```

Comment:
I think this could go in CatalogueItemDetails as its common, but then again it would really be the ID, which isn't in yet so I would move it in that instead.